rfc/rfc004-verifiable-transactional-graph.md

```diff
@@ -66,16 +66,15 @@ In addition to the registered header parameters, the following headers MUST be p
 * **prevs**: \(previous transactions\) MUST contain references \(see section 3.2\) of preceding transactions \(see section 3.4\).
   * When it's a root transaction the field SHALL NOT have any entries.
   * When creating a transaction it MUST only contain transactions that the local node has successfully processed, to avoid publishing unprocessable transactions.
-* **lc**: MUST be a positive number constructed as follows:
-  * if the transaction has no entries in **prevs**, it's' lc value is **0**.
-  * otherwise, the value MUST be equal to `max(prev1, ... prevN)+1`.
+* **lc**: (Lamport's Logical Clock) MUST be a natural number.
+  * If the transaction has **prevs** entries, then **lc** is 0.
+  * Otherwise, **lc** MUST be the highest **prevs** entry plus 1.
 
 The following protected headers MAY be present:
 
 * **pal**: MUST contain the encrypted addresses of the participants \(used for private transactions, see section 3.8\).
 
-When adding a new transaction, that transaction MUST point (through the **prevs** field) to the transaction with the highest **lc** value.
-If multiple transactions have the highest **lc** value, a single one of them SHOULD be used.
+For a DAG, any and all transaction additions MUST point to the transaction in with the highest **lc** value present. When multiple transaction additions share the same **lc** value, then only a single one of them SHOULD be used. Note that any transaction omitted due to **lc** duplicates needs a retry, or be lost.
```

Comment:
incorrect, especially `Note that any transaction omitted due to **lc** duplicates needs a retry, or be lost.` doesn't make any sense.

Reply:
No idea what I was thinking there. 😆